recipes/nagios/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "nagios" %}
+{% set version = "4.4.7" %}
+{% set sha256 = "6429d93cc7db688bc529519a020cad648dc55b5eff7e258994f21c83fbf16c4d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/NagiosEnterprises/nagioscore/releases/download/{{ name }}-{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+  script:
+    - ./configure --prefix $PREFIX --with-nagios-user=`whoami` --with-nagios-group=`whoami`  # [unix]
+    - ./configure --prefix $PREFIX --with-nagios-user=`whoami` --with-nagios-group="staff"  # [osx]
+    - make all
+    - make install
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make  # [not win]
+    - openssl
+  host:
+    - openssl
+    - unzip  # [unix] 
+    - pkg-config  # [unix] 
+  run: []
+
+test:
+  commands:
+    - $PREFIX/bin/nagios --version
```

```suggestion
    - nagios --version
```
$PREFIX is the packaging prefix and it will not exist on the end users machine. The end user will have the package installed in $CONDA_PREFIX.